sdk/highlight-node/src/client.ts

```diff
@@ -110,11 +111,12 @@ export class Highlight {
 		name: string,
 		value: number,
 		requestId?: string,
-		tags?: { name: string; value: string }[],
```

Comment:
Do we need to mention anything about `tags` being removed on `recordMetric`? Also, are there some docs we should update?

Reply:
you're right - this is a breaking change changing the types like that so i'll refrain from making it here. we'll likely be changing the API for metrics more as we get to that part of the product.